### ISSUE TITLE
extra dash for kubecontext zone info

### ISSUE THREAD

**Issue by yinanfang** [romkatv/powerlevel10k#1013]
I'm using the most recent (v1.13.0) theme and [classic config](https://github.com/romkatv/powerlevel10k/blob/master/config/p10k-classic.zsh) with the following overwrites for kubecontext:

```
typeset -g POWERLEVEL9K_KUBECONTEXT_DEFAULT_CONTENT_EXPANSION=
POWERLEVEL9K_KUBECONTEXT_DEFAULT_CONTENT_EXPANSION+='${${:-/$P9K_KUBECONTEXT_CLOUD_NAME}:#/default}'
POWERLEVEL9K_KUBECONTEXT_DEFAULT_CONTENT_EXPANSION+='${${:-/$P9K_KUBECONTEXT_CLOUD_ACCOUNT}:#/default}'
POWERLEVEL9K_KUBECONTEXT_DEFAULT_CONTENT_EXPANSION+='${${:-/$P9K_KUBECONTEXT_CLOUD_CLUSTER}:#/default}'
POWERLEVEL9K_KUBECONTEXT_DEFAULT_CONTENT_EXPANSION+='${${:-/$P9K_KUBECONTEXT_CLOUD_ZONE}:#/default}'
POWERLEVEL9K_KUBECONTEXT_DEFAULT_CONTENT_EXPANSION+='${${:-/$P9K_KUBECONTEXT_NAMESPACE}:#/default}'
```

It's basically showing all cluster info for the kubecontext. However, I notice that the the zone display has some issue. I think something in the string processing for the zone info is wrong. This is what I got for `Show on command` feature in the terminal:

```
~/Code/MyCode                   /gke/project/cluster/us-central1--f ⎈
```

**Comment by romkatv** [comment]
Thanks for the report! Fixed.

P.S.

Your config looks somewhat odd. It would make sense if `P9K_KUBECONTEXT_CLOUD_NAME` could be equal to "default" but it never is. Try this instead:

```zsh
typeset -g POWERLEVEL9K_KUBECONTEXT_DEFAULT_CONTENT_EXPANSION='${${P9K_KUBECONTEXT_CLOUD_NAME:+/$P9K_KUBECONTEXT_CLOUD_NAME/$P9K_KUBECONTEXT_CLOUD_ACCOUNT/$P9K_KUBECONTEXT_CLOUD_CLUSTER/$P9K_KUBECONTEXT_CLOUD_ZONE}:-$P9K_KUBECONTEXT_NAME}${${:-/$P9K_KUBECONTEXT_NAMESPACE}:#/default}'
```

In plain English: If `P9K_KUBECONTEXT_CLOUD_NAME` is set, show `/cloud/account/cluster/zone/namespace`; otherwise show `name/namespace`. In both cases, don't show `/namespace` if it's literally `/default`.

**Comment by yinanfang** [comment]
@romkatv pulled latest and works great. Thanks for the quick fix.
Yeah your config is better.
btw, this theme is awesome!

**Comment by romkatv** [comment]
> Maybe we can use that as default?

It way too verbose for a default.

> btw, this theme is awesome!

Thanks!
